playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -71,20 +71,14 @@
 - hosts: localhost
   gather_facts: no
   tasks:
-  - name: get current date/time
-    shell: TZ=Etc/UTC date +%Y%m%d%H%M
```

Comment:
looks good, can you make the same change in build_base_image.yml

Reply:
done